File: official/docs/python/v7/batches/add-shipments.py
import easypost

easypost.api_key = "EASYPOST_API_KEY"

batch = easypost.Batch.retrieve("batch_...")

batch.add_shipments(
    shipments=[
        {"id": "shp_..."},
        {"id": "shp_..."},
    ]
)

print(batch)

File: official/docs/php/v5/batches/add-shipments.php
<?php

\EasyPost\EasyPost::setApiKey('EASYPOST_API_KEY');

$batch = \EasyPost\Batch::retrieve('batch_...');

$batch->add_shipments([
    'shipments' => [
        ['id' => 'shp_...'],
        ['id' => 'shp_...'],
    ]
]);

echo $batch;
